frontend/src/components/student/AllActivities/CombatTask/CombatTask.js

```diff
@@ -81,69 +78,131 @@ export default function CombatTask() {
     []
   )
 
-  return (
-    <Content>
-      <InfoContainer fluid className='p-0'>
-        {!task && errorMessage === '' ? (
-          <Loader />
-        ) : errorMessage !== '' ? (
-          <p>{errorMessage}</p>
-        ) : (
-          <ActivityCol className='invisible-scroll'>
-            <HeaderCol>
-              <HeaderRow>
-                <ActivityImg src={getActivityImg('TASK')}></ActivityImg>
-                <ActivityType>{getActivityTypeName('TASK')}</ActivityType>
-                <ActivityName>{task.name}</ActivityName>
-              </HeaderRow>
-              <FullDivider />
-            </HeaderCol>
-            <div>
-              <h5>{task.description}</h5>
-              {task.points !== null && (
+  //add overflows where needed
+
+  const Header = () => (
+    <>
+      <ActivityImg src={getActivityImg('TASK')}></ActivityImg>
+      <ActivityType>{getActivityTypeName('TASK')}</ActivityType>
+      <ActivityName>{task.name}</ActivityName>
+    </>
+  )
+
+  const VerticalSpacer = () => <Row style={{ height: '2vh' }}></Row>
+
+  const HorizontalSpacer = () => <Col style={{ height: '3vh' }} />
+
+  const ActivityDetails = () => (
+    <Col>
+      <h2>Treść:</h2>
+      <p>{task.description}</p>
+    </Col>
+  )
+
+  const AwaitingFeedbackField = () => (
+    <Col md={6} className={'text-center p-4 my-auto'}>
+      <FontAwesomeIcon className={'m-2'} icon={faHourglass} size='5x' spin />
+      <h2 className={'m-2'}>Odpowiedź została przesłana, oczekiwanie na sprawdzenie przez prowadzącego</h2>
+    </Col>
+  )
+
+  const content = () => {
+    return !task && errorMessage === '' ? (
+      <Loader />
+    ) : errorMessage !== '' ? (
+      <p>{errorMessage}</p>
```

Comment:
Wiem, że ten `errorMessage` to moja robota, ale już utarła nam się konwencja z nullem gdy error (wiesz chyba jaka 😉 ). Więc tutaj bym też tak robił jak wszędzie indziej robimy